docs/workflow/building/coreclr/linux-instructions.md

```diff
@@ -87,8 +87,9 @@ After the build is completed, there should some files placed in `artifacts/bin/c
 Create the Core_Root
 ===================
 
-The Core_Root folder will contain the built binaries, generated by `build.sh`, as well as the library packages required to run tests. Note that you need to build
-the libraries subset (`-subset libs`) before this command can be run.
+The Core_Root folder will contain the built binaries, generated by `build.sh`, as well as the library packages required to run tests. It is required that you build
```

Comment:
I just wonder if https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/testing.md should be the place that has this information. We keep this info so fragmented, that one place is always wrong.